civiform/civiform
# [Program Card Images] Add the generic image icon and use it for the "Edit program image" button.

### Description

This PR adds a generic image icon and uses it for the "Edit program image" button.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Created unit and/or browser tests which fail without the change (if possible) -- **existing tests are sufficient**

### Instructions for manual testing

1. Enable the PROGRAM_CARD_IMAGES flag.
2. As an admin, edit a program.
3. Verify the program edit homepage shows a generic image icon in the "Edit program image" button.

### Issue(s) this completes

Epic #5676
